### PR TITLE
Copsey

### PR DESCRIPTION
**Pull request by simoncopsey**
Distilled 'why you' section for all job specs into 'responsibilities' and 'qualifications' sections, adding more tangibles.
Modified 'why us' to further speak to the culture.

### CODE REVIEW

File: Recruitment/JobSpecs/why_us.md [hunk 1]
```diff
@@ -1,34 +1,17 @@
...
-
-We’re looking for people who want to be treated like adults – who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it. We’re looking for people who are comfortable finding their own place.
-
-We’re looking for people who are willing to seek input when they want to make a decision that will affect others, or when others know more than they do.
 
-We build software. We build teams. We build new ways of working. We build better organisations.
+### Our approach
+We’re looking for people who want to help us change the way companies think and approach problems; people who want to work individuals and cross-functional teams who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.
```
Comment (yannislionis) [inline Recruitment/JobSpecs/why_us.md]: I think the grammar in the "people who want to work..." needs some adjustments

File: Recruitment/JobSpecs/why_us.md [hunk 2]
```diff
@@ -1,34 +1,17 @@
...
 
-###Our promise
-We provide dedicated time and support for you to own your own development and growth.
+We’re value polygots who neither need, nor want, to be told what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it. We’re looking for people who are comfortable finding their own place in a flat organisation amongst egoless peers who are willing to listen, learn, and share.
```
Comment (yannislionis) [inline Recruitment/JobSpecs/why_us.md]: "We value" rather?

File: Recruitment/JobSpecs/why_us.md [hunk 3]
```diff
@@ -1,34 +1,17 @@
...
-
-We’re looking for people who want to be treated like adults – who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it. We’re looking for people who are comfortable finding their own place.
-
-We’re looking for people who are willing to seek input when they want to make a decision that will affect others, or when others know more than they do.
 
-We build software. We build teams. We build new ways of working. We build better organisations.
+### Our approach
+We are looking for people who want to help us change the way companies think and approach problems; people who want to work as individuals and cross-functional teams, who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.
```
Comment (shahzainb) [inline Recruitment/JobSpecs/why_us.md]: This sentence is still very complicated and you lose the trend of thought part way through. We need to simplify the emotions here IMO.

File: Recruitment/JobSpecs/why_us.md [hunk 4]
```diff
@@ -1,34 +1,17 @@
...
 
-###Our promise
-We provide dedicated time and support for you to own your own development and growth.
+We are value polygots who neither need, nor want, to be told what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it. We’re looking for people who are comfortable finding their own place in a flat organisation amongst egoless peers who are willing to listen, learn, and share.
 
-Your contributions to our community will be valued and promoted.
+We are genuinely curious about business, UX, strategy, social and behavioural factors and appreciate that engineering is a key piece of a bigger puzzle, understanding the other pieces and ‘the whole’. We enjoy finding ways to make complicated tasks easy, elminating things that aren’t valuable and automating things that are. We can often be heard saying “it’s good enough” and “let’s ship it”, but also “what is our goal?” and “this should be nowhere near production”. We encourage everyone to bring their personality to work and to seek input when they want to make a decision that will affect others, or when others know more than they do.
```
Comment (shahzainb) [inline Recruitment/JobSpecs/why_us.md]: Same comment as above.

File: Recruitment/JobSpecs/why_us.md (hunk 3 above)
Comment (simoncopsey) [inline Recruitment/JobSpecs/why_us.md]: Thanks for the catches and suggestions! Have reworded accordingly - and removed the typos.

File: Recruitment/JobSpecs/why_us.md (hunk 2 above)
Comment (simoncopsey) [inline Recruitment/JobSpecs/why_us.md]: Thanks for the catches - have reworded and corrected the typos!

File: Recruitment/JobSpecs/why_you_js_engineer.md [hunk 5]
```diff
@@ -1,43 +1,19 @@
-### Why You?
+### What you'll do
+* Work intimiately with clients to understand their organisation, aspirations and challenges.
```
Comment (shahzainb) [inline Recruitment/JobSpecs/why_you_js_engineer.md]: there's a spelling mistake here mate. find it yourself.

File: Recruitment/JobSpecs/why_you_js_engineer.md [hunk 6]
```diff
@@ -1,43 +1,19 @@
...
+* Help clients identify promising solutions through participating in, and leading, digital workshops.
+* Build real solutions through experimentation as part of a multi-faceted team.  
+* Accelerate and improve how we help our clients through developing and contributing to components, services and methodologies.
```
Comment (shahzainb) [inline Recruitment/JobSpecs/why_you_js_engineer.md]: So maybe we need to say something like "we do great new things, using great new tech, but we don't always work with clients, lots of cool things to build internally as well. and contribute back to the community. peace."

File: Recruitment/JobSpecs/why_you_js_engineer.md [hunk 7]
```diff
@@ -1,43 +1,19 @@
...
+* Explore up-and-coming technologies and software products; seize learning and knowledge sharing opportunities; participate in professional organizations. Read; teach; learn.
 
-Our clients know that there's a better way to deliver software, but aren't always sure what it is, or how to get there. When they are ready to change, we help them.
 
-If that sounds like something you'd like to be a part of, here are some things to consider:
+### What we value
+* Are you a talented and passionate Javascript Engineer, with experience in handling both server-side and client-side code?
```
Comment (shahzainb) [inline Recruitment/JobSpecs/why_you_js_engineer.md]: maybe remove the questions and change tone to statements?

**Pull request merged**